applications/GeoMechanicsApplication/custom_processes/apply_constant_phreatic_multi_line_pressure_process.cpp

```diff
@@ -81,6 +81,13 @@ void ApplyConstantPhreaticMultiLinePressureProcess::ValidateCoordinates(const Pa
     if (!std::is_sorted(HorizontalDirectionCoordinates().begin(), HorizontalDirectionCoordinates().end())) {
         KRATOS_ERROR << "The Horizontal Elements Coordinates are not ordered." << rParameters << std::endl;
     }
```

Comment:
What do you think, perhaps, it is nice to use KRATOS_ERROR_IF here and above? Everything will be in the same style. Please, put is_sorted check below next checks.